tests/test_all.py

```diff
@@ -435,34 +440,8 @@ def test_certificates_are_present(
 
     If the certificates are incorrectly set up, then the GET request will fail.
     """
-    multi_stage_build = MultiStageBuild(
-        containers={
-            "builder": "registry.suse.com/bci/golang:latest",
-            "runner": runner,
-        },
-        containerfile_template=MULTISTAGE_DOCKERFILE,
-    )
-    multi_stage_build.prepare_build(
-        tmp_path, container_runtime, pytestconfig.rootpath
-    )
-
-    with open(tmp_path / "main.go", "w", encoding="utf-8") as main_go:
-        main_go.write(FETCH_SUSE_DOT_COM)
-
-    # FIXME: ugly duplication of pytest_container internals :-/
-    # see: https://github.com/dcermak/pytest_container/issues/149
-    iidfile = tmp_path / "iid"
-    host.run_expect(
-        [0],
-        f"{' '.join(container_runtime.build_command + get_extra_build_args(pytestconfig))} "
-        f"--iidfile={iidfile} {tmp_path}",
-    )
-    img_id = container_runtime.get_image_id_from_iidfile(iidfile)
-
-    host.run_expect(
-        [0],
-        f"{container_runtime.runner_binary} run --rm {' '.join(get_extra_run_args(pytestconfig))} {img_id}",
-    )
+    out = host.check_output(container_image.run_command)
+    assert "html" in out
```

Comment:
nit: I'm not sure if this is useful, checking for a real HTML tag found on the page would be more worthwhile.

`<link rel="canonical" href="https://www.suse.com/" />`